src/components/RecipientsComponent.jsx

```diff
@@ -82,35 +84,55 @@ class RecipientsComponent extends Component {
               <Grid container justify='space-between'>
                 <Grid item>
                   <Box display='flex' alignItems='center'>
-                    <UserAvatar
-                      style={{ width: 32 }}
-                      name={recipient.name}
-                      src={recipient.imageUrl}
-                    />
-                    <Box ml={1}>
-                      <Typography data-test-id='recipient_name'>{recipient.name}</Typography>
-                      <Typography variant='caption' data-test-id='recipient_email'>
-                        {recipient.email}
-                      </Typography>
-                    </Box>
+                    {!skeletonOnly ? (
+                      <UserAvatar
+                        style={{ width: 32 }}
+                        name={recipient.name}
+                        src={recipient.imageUrl}
+                      />
+                    ) : (
+                      <Skeleton
+                        variant='circle'
+                        width={32}
```

Comment:
might well just put width and height to class too

Reply:
this is circle, different from other skeleton